controllers/synapse/synapse_configmap.go

```diff
@@ -1122,7 +1122,7 @@ url_preview_accept_language:
 
 # Enable registration for new users.
 #
-#enable_registration: false
+enable_registration: true
```

Comment:
This change seems unrelated to the rest of the code. What is it for? Could be done in a separate commit with some comments to clarify what is happening here.

Reply:
I moved it to a separate commit